cmd/goquery_completion/conditional.go

```diff
@@ -31,6 +31,86 @@ func openParens(tokens []string) int {
 	return open
 }
 
+// dirKeywordCount returns the number of dir keyword occurrences in tokens
+func dirKeywordCount(tokens []string) int {
```

Comment:
As hinted in the comment in the draft (sorry, didn't see it was still in that state, I just did what was asked in the notification and blindly reviewed :stuck_out_tongue: ), these counting functions are prime candidates for named return parameters, because it unclutters the code (and removes the need to manually initialize a local variable). Also, IMHO if the (short) function is already clearly named and just has a single counter, the variable name can easily be shortened (e.g. to `count` in this case):
```
func dirKeywordCount(tokens []string) (count int) {
	for _, token := range tokens {
		if token == types.FilterKeywordDirection || token == types.FilterKeywordDirectionSugared {
			count++
		}
	}
	return
}
```